### PR TITLE
Feature/agendapunten rework

### PR DESCRIPTION
**Pull request by abeforgit**
Complete rework of the agenda management components

Key points: 
- item list is rendered directly in the page, aka one modal less
- items are saved individually (no more store-and-save-later)
- thorough rewrite of the code
fixes:
https://binnenland.atlassian.net/browse/GN-2198

### CODE REVIEW

**Comment by Dietr** [comment]
Looks good. Much more in line with the other modals on the meeting form page. I added some styling and removed some old webuniversum classes.

**Review by Dietr — APPROVED** [review]

File: app/templates/imported-documents/publish.hbs [hunk 1]
```diff
@@ -277,7 +277,7 @@
                       <li>
                         <a class="popover__link" href="#">Verklaar ondertekeningen nietig</a>
                         Need modal for this one, hide if too much for now
-                        <p class="smaller u-spacer">U kan dit document niet meer nietig verklaren, omdat de aanvullende agenda al gepubliceerd werd.</p> wss niet nodig
+                        <p class="smaller u-spacer">U kan dit document niet meer nietig verklaren, omdat de aanvullende agenda-manager al gepubliceerd werd.</p> wss niet nodig
```
Comment (nvdk) [inline app/templates/imported-documents/publish.hbs:280]: woops, replaced a string too many ;)

File: app/templates/login.hbs [hunk 2]
```diff
@@ -35,7 +35,7 @@
               <p>Notuleer uw zitting met behulp van het online tekstverwerkingsprogramma.</p>
               <ul>
                 {{!--
-                <li>stel <strong>agenda's</strong> op, bereid <strong>agendapunten</strong> voor en <strong>notuleer</strong> uw zitting;</li>
+                <li>stel <strong>agenda-manager's</strong> op, bereid <strong>agendapunten</strong> voor en <strong>notuleer</strong> uw zitting;</li>
```
Comment (nvdk) [inline app/templates/login.hbs:38]: remove manager here

File: app/components/document-link-to-published.js [hunk 3]
```diff
@@ -30,8 +30,8 @@ export default Component.extend({
         // concept
         return null;
       case "627aec5d144c422bbd1077022c9b45d1":
-        // agenda publiek
-        return "Bekijk&nbsp;publieke&nbsp;agenda";
+        // agenda-manager publiek
+        return "Bekijk&nbsp;publieke&nbsp;agenda-manager";
```
Comment (nvdk) [inline app/components/document-link-to-published.js:34]: remove manager here

**Review by nvdk — CHANGES_REQUESTED** [review]

File: app/components/agenda-manager/agenda-context.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    item.titel = "";
+    item.beschrijving = "";
+    item.geplandOpenbaar = true;
+    item.behandeling = this.createBehandeling(item);
+    return item;
+  }
+
+  @action
+  createBehandeling(agendapunt) {
+    const documentContainer = this.store.createRecord("document-container");
```
Comment (nvdk) [inline app/components/agenda-manager/agenda-context.js:108]: document container needs to have a folder and status set

**Review by nvdk — APPROVED** [review]

**Pull request merged**